lib/utils/protocols/rtp.ts

```diff
@@ -66,7 +66,7 @@ export const cSrc = (buffer: Buffer, rank = 0) => {
 }
 
 export const extHeaderLength = (buffer: Buffer) => {
-  return extension(buffer) === false
+  return !extension(buffer)
```

Comment:
These are not the same. Unless you are sure `extension(buffer)` will only return `true` or `false`.

Reply:
I wonder if this was autofixed. It should only be the same if it returns a boolean. I'll double check.

Reply:
It's supposed to be a boolean, so the change is fine.